core/src/main/java/org/apache/druid/data/input/InputFormat.java

```diff
@@ -41,15 +41,17 @@
  * See {@link NestedInputFormat} for nested input formats such as JSON.
  */
 @UnstableApi
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = InputFormat.TYPE_PROPERTY)
 @JsonSubTypes(value = {
-    @Type(name = "csv", value = CsvInputFormat.class),
-    @Type(name = "json", value = JsonInputFormat.class),
+    @Type(name = CsvInputFormat.TYPE_KEY, value = CsvInputFormat.class),
+    @Type(name = JsonInputFormat.TYPE_KEY, value = JsonInputFormat.class),
     @Type(name = "regex", value = RegexInputFormat.class),
```

Comment:
Since other string constants are turned into a static constant in each class, why does this one leave unchanged?